frontend/src/actions/profileActions.js

```diff
@@ -107,6 +137,102 @@ export const getFriendProfileByURL = (url) => (dispatch, getState) => {
         });
 }
 
+// DELETE friend by url
+export const deleteFriendByURL = (author_url, friend_url) => (dispatch, getState) => {
```

Comment:
Usually format operation happened inside component functions that bind to correspond component. 

actions dont do those js logics or parsing. This is fine for now, try to avoid doing parsing in actions in the future